base/exports.jl

```diff
@@ -23,6 +23,7 @@ export
     Docs,
     Markdown,
     Threads,
+    Iterators,
```

Comment:
it looks like this is now confusing precompile because it finds the `Main.Iterators` binding (probably using the wrong logic) and ends up having that shadow the real Iterators package. This shows up as breakage of the Compose package:

```julia
julia> using Compose
WARNING: The call to compilecache failed to create a usable precompiled cache file for module Compose. Got:
WARNING: Module Iterators uuid did not match cache file.

julia> using Iterators
WARNING: imported binding for Iterators overwritten in module Main

julia> using Compose

julia> 
```